### PR TITLE
Detailslist focusable

### PR DESCRIPTION
**Pull request by smhigley**
## Previous Behavior

Related to #25342 -- that PR fixed this same issue for when the header row is entirely absent. The issue still presents when the DetailsList header is present but not focusable (i.e. by disabling both selection and `columnActionsMode`).

## New Behavior

The first row of the DetailsList body is now always in the tab order. When I was taking another look at the earlier fix, I realized that there's no use case when we _wouldn't_ want the first content row to be in the tab order, since the headers are a separate tab stop from the body.

## Related Issue(s)

- Fixes [ADO bug](https://dev.azure.com/microsoftdesign/fluent-ui/_workitems/edit/18499)

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/be3a3d59d5c30f75d55c739548d576594ddf5c53'>be3a3d59d5c30f75d55c739548d576594ddf5c53</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>BaseButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/BaseButton-mount_base.html">619</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/BaseButton-mount.html">623</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Breadcrumb</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/Breadcrumb-mount_base.html">1662</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/Breadcrumb-mount.html">1682</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Checkbox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/Checkbox-mount_base.html">1685</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/Checkbox-mount.html">1720</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>CheckboxBase</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/CheckboxBase-mount_base.html">1505</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/CheckboxBase-mount.html">1473</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ChoiceGroup</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/ChoiceGroup-mount_base.html">2957</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/ChoiceGroup-mount.html">2995</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ComboBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/ComboBox-mount_base.html">648</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/ComboBox-mount.html">645</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>CommandBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/CommandBar-mount_base.html">6216</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/CommandBar-mount.html">6201</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/ContextualMenu-mount_base.html">12546</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/ContextualMenu-mount.html">13473</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DefaultButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/DefaultButton-mount_base.html">738</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/DefaultButton-mount.html">747</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRow</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/DetailsRow-mount_base.html">2193</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/DetailsRow-mount.html">2207</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowFast</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/DetailsRowFast-mount_base.html">2164</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/DetailsRowFast-mount.html">2188</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowNoStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/DetailsRowNoStyles-mount_base.html">2035</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/DetailsRowNoStyles-mount.html">2024</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Dialog</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/Dialog-mount_base.html">2677</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/Dialog-mount.html">2673</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DocumentCardTitle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/DocumentCardTitle-mount_base.html">230</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/DocumentCardTitle-mount.html">225</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Dropdown</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/Dropdown-mount_base.html">1973</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/Dropdown-mount.html">2008</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusTrapZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/FocusTrapZone-mount_base.html">1141</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/FocusTrapZone-mount.html">1138</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/FocusZone-mount_base.html">1097</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/FocusZone-mount.html">1069</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/GroupedList-mount_base.html">41755</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/GroupedList-mount.html">41726</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/GroupedList-virtual-rerender_base.html">20056</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/GroupedList-virtual-rerender.html">20101</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/GroupedList-virtual-rerender-with-unmount_base.html">50388</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/GroupedList-virtual-rerender-with-unmount.html">50421</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/GroupedListV2-mount_base.html">234</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/GroupedListV2-mount.html">229</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/GroupedListV2-virtual-rerender_base.html">205</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/GroupedListV2-virtual-rerender.html">208</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/GroupedListV2-virtual-rerender-with-unmount_base.html">224</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/GroupedListV2-virtual-rerender-with-unmount.html">228</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>IconButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/IconButton-mount_base.html">1078</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/IconButton-mount.html">1096</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Label</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/Label-mount_base.html">327</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/Label-mount.html">338</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Layer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/Layer-mount_base.html">2696</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/Layer-mount.html">2693</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Link</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/Link-mount_base.html">396</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/Link-mount.html">384</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MenuButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/MenuButton-mount_base.html">945</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/MenuButton-mount.html">952</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MessageBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/MessageBar-mount_base.html">21177</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/MessageBar-mount.html">22268</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Nav</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/Nav-mount_base.html">1936</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/Nav-mount.html">1961</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>OverflowSet</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/OverflowSet-mount_base.html">785</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/OverflowSet-mount.html">802</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Panel</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/Panel-mount_base.html">1788</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/Panel-mount.html">1781</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/Persona-mount_base.html">761</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/Persona-mount.html">768</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Pivot</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/Pivot-mount_base.html">874</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/Pivot-mount.html">877</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>PrimaryButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/PrimaryButton-mount_base.html">828</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/PrimaryButton-mount.html">870</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Rating</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/Rating-mount_base.html">4670</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/Rating-mount.html">4605</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SearchBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/SearchBox-mount_base.html">924</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/SearchBox-mount.html">915</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Shimmer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/Shimmer-mount_base.html">1868</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/Shimmer-mount.html">1862</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Slider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/Slider-mount_base.html">1326</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/Slider-mount.html">1349</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/SpinButton-mount_base.html">2860</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/SpinButton-mount.html">2912</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Spinner</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/Spinner-mount_base.html">383</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/Spinner-mount.html">394</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SplitButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/SplitButton-mount_base.html">1800</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/SplitButton-mount.html">1802</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Stack</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/Stack-mount_base.html">399</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/Stack-mount.html">406</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithIntrinsicChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/StackWithIntrinsicChildren-mount_base.html">854</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/StackWithIntrinsicChildren-mount.html">858</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithTextChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/StackWithTextChildren-mount_base.html">2594</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/StackWithTextChildren-mount.html">2603</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchColorPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/SwatchColorPicker-mount_base.html">6214</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/SwatchColorPicker-mount.html">6179</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TagPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/TagPicker-mount_base.html">1461</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/TagPicker-mount.html">1496</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Text</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/Text-mount_base.html">373</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/Text-mount.html">372</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TextField</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/TextField-mount_base.html">953</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/TextField-mount.html">939</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/ThemeProvider-mount_base.html">821</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/ThemeProvider-mount.html">822</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/ThemeProvider-virtual-rerender_base.html">589</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/ThemeProvider-virtual-rerender.html">580</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/ThemeProvider-virtual-rerender-with-unmount_base.html">1299</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/ThemeProvider-virtual-rerender-with-unmount.html">1280</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Toggle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/Toggle-mount_base.html">616</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/Toggle-mount.html">620</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>buttonNative</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/buttonNative-mount_base.html">187</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30386/perf-test/buttonNative-mount.html">190</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9423"/>

File: packages/react/src/components/DetailsList/DetailsList.base.tsx [hunk 1]
```diff
@@ -495,8 +495,7 @@ const DetailsListInner: React.ComponentType<IDetailsListInnerProps> = (
       const rowRole = role === defaultRole ? undefined : 'presentation';
 
       // add tabindex="0" to first row if no header exists, to ensure the focuszone is in the tab order
-      const rowFocusZoneProps =
-        isHeaderVisible || index > 0 ? rowFocusZoneNoTabIndexProps : rowFocusZoneAddTabIndexProps;
+      const rowFocusZoneProps = index > 0 ? rowFocusZoneNoTabIndexProps : rowFocusZoneAddTabIndexProps;
```
Comment (spmonahan) [inline packages/react/src/components/DetailsList/DetailsList.base.tsx:498]: Is the comment above still accurate?

Comment (smhigley) [reply packages/react/src/components/DetailsList/DetailsList.base.tsx:498]: Ah good catch, I'll update it

**Review by spmonahan — APPROVED** [review]

**Pull request merged**